include/rocksdb/utilities/options_util.h

```diff
@@ -58,7 +58,9 @@ namespace rocksdb {
 Status LoadLatestOptions(const std::string& dbpath, Env* env,
                          DBOptions* db_options,
                          std::vector<ColumnFamilyDescriptor>* cf_descs,
-                         bool ignore_unknown_options = false);
+                         bool ignore_unknown_options = false,
+                         std::shared_ptr<Cache> cache = {}
```

Comment:
I think the current code uses in the way of `std::shard_ptr<Cache>*`: https://github.com/facebook/rocksdb/blob/master/include/rocksdb/env.h#L370-L371
I suggest we follow it.